packages/relayer/src/logger.rs

```diff
@@ -0,0 +1,43 @@
+use file_rotate::{
+    compression::Compression,
+    suffix::{AppendTimestamp, FileLimit},
+    ContentLimit, FileRotate,
+};
+use lazy_static::lazy_static;
+use slog::{o, Drain, Level};
+use slog_async;
+use slog_json;
+use slog_term;
+
+lazy_static! {
+    pub static ref LOG: slog::Logger = init_logger();
+}
+
+fn init_logger() -> slog::Logger {
+    let log_terminal_decorator = slog_term::TermDecorator::new().build();
+    let log_terminal_drain = slog_term::FullFormat::new(log_terminal_decorator)
+        .build()
+        .fuse();
+
+    let directory = std::path::Path::new("logs");
+    let log_path = directory.join("relayer.log");
+    let file_rotate = FileRotate::new(
+        log_path.clone(),
+        AppendTimestamp::default(FileLimit::MaxFiles(1_000_000)),
+        ContentLimit::Bytes(5_000_000),
+        Compression::None,
```

Comment:
If you like, could you please change this line to 
like `Compression::OnRotate(5)` or something? 
because in the current implementation, the docker container uses disk space indefinitely.

Reply:
Implemented.